section1/reflection.md

```diff
@@ -1,19 +1,28 @@
 
 2. What are the data types you learned about in this section? In your own words, define each.
+We learned about strings, number, and boolean data types. A string is a data type that consists of letters and any characters wrapped in quotes. A number data type that consists of exclusively numbers. While it can encompass negatives and decimals it must be a set of numbers. A Boolean data type evaluates two values on a true/false basis. The yes or no type of result it provides can be decided by different rules, parameters, and operators influencing it.
 
 3. How would you log the string `"Hello World!"` to the console?
-
+console.log("Hello World")
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+// Anything after two backslashes for a shorter in line comments
+/*" "*/ Anything in between the backslashes and asterisks for more extensive multi line comments.
 
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+I variable is something that can store values to use in different places. That variable is assigned and can be reassigned if you need different values. It can be used to store simplistic information or complex portions of code to easily use later.
 
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+Reddit.com  When you decide to make a post it most likely uses your data from stored variables. A username on a post would be a string data type. Your birthday information provided on the post for that silly cake-day thing would be a number data type. When choosing to view content that is not safe work they ask if you want to proceed. That yes or no variable would be a Boolean, either True/False.
 
 7. In your own words, explain what concatenation is.
+A concatenation is the process of adding strings and values together though the "+" operator. JavaScript can merge a string and number variable together through the use of Coercion. Coercion is simply converting data types from one to another based off JavaScripts parameters.
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+Im assuming that developers would use concatenation for smaller strings such as a welcome notice when you re log into their site. It gets complex to use that many "+" operators when you are typing something out thats longer and I assume it would only be used for quick strings.
 
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+I have question about how to properly use a Boolean variable. I found it slightly confusing and feel like there are better ways to utilize the True/False results. Im guessing we just have not gotten to explore it enough yet, but I felt it was difficult in the section 1 exercises to use it effectively.
```

Comment:
Yeah, good question! Hopefully you feel more confident from working through section 2 & 3, but rest assured that you'll get plenty more practice as you start to build programs and apps with complex logic. Every decision can be considered a True/False evaluation!